src/main/java/com/google/sps/data/ScheduledInterview.java

```diff
@@ -0,0 +1,80 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps;
+
+import java.time.LocalDate;
+
+/** Represents a scheduled interview. */
+public class ScheduledInterview {
```

Comment:
AutoValue?

Copy-paste of my comment from https://github.com/googleinterns/InterviewMe/pull/43/files but updated for your use case

---

https://github.com/google/auto/tree/master/value

Here's what the current version of the code might look like with autovalue:
```java
  import com.google.auto.value.AutoValue;

  @AutoValue
  abstract static class ScheduledInterview {
    abstract TimeRange when();
    abstract LocalDate date();
    abstract String interviewerEmail();
    abstract String intervieweeEmail();

    static ScheduledInterview create(TimeRange when, LocalTime date, String interviewerEmail, String intervieweeEmail) {
      return new AutoValue_ScheduledInterview(when, date, interviewerEmail, intervieweeEmail);
    }
  }
```
I can give the pod a primer on AutoValue if would be helpful. I find it incredibly useful both to simplify code, and also as a tool for thinking about problems, since I find the best code strongly distinguishes between simple value types (i.e. above), and classes which do lots of work using value types (i.e. a servlet class).